### ISSUE TITLE
add README file

### ISSUE THREAD

**Issue by PeterTheOne** [piratar/parli#1]
What is parli in relation to wasa2il? Is it a complete rewrite?

**Comment by bjornlevi** [comment]
Yes. Some code will be reused of course but wasa2il was built in the old django structure and has a lot of unneeded complexity/components. Unwinding wasa2il is just as much work as building it up from scratch. It's not "that" much work anyway.

**Comment by PeterTheOne** [comment]
Thanks for the info. I don't know about the feature set of wasa2il. Will this support preferential/ranked voting and delegations (as in liquid democracy)?

**Comment by bjornlevi** [comment]
Yes, the plan is to provide a better plugin system for vote results.

**Comment by valberg** [comment]
Sorry if I'm hijacking this issue, but since there is already a discussion going on I would like to pitch in.

I think this is a great idea. I've been playing around with wasa2il, which is cool and all, but could use some refactoring :smile: 

I propose that parli is kept:
- Organisation agnostic: wasa2il has the Píratar logo, not a big problem, but nice to think of from the beginning)
- Language agnostic: use gettext for all strings and keep them in english and do translations
- Database agnostic: wasa2il has MySQL python libraries in requirements.txt, which should be possible to avoid due to the awesomeness of the Django ORM.

I'm totally up for helping with what I can since this is such an interesting and powerful way of organising movements and people in general :+1:

**Comment by bjornlevi** [comment]
Yeah, have to set up my editor to convert tabs to spaces. My bad - have it on my work comp.

wasa2il is language agnostic, I've got to step up my internationalization :) --- in some cases it is better to internationalize afterwards though. Faster development up front and then a specific internationalization task after. It's more "focused".

Parli is out of the box sqlite3.

Good point on the organization. Although that's really just changing the base.html to get a new org. But still.

**Comment by valberg** [comment]
@bjornlevi Yeah those things are easy to achieve, just wanted to put them out there so it's been thought about from the beginning :smile: 

Do you need any help at the moment, or do you need some more time to flesh out the current code?

If we get some issues up for different tasks we can begin parallel work :smile:

**Comment by PeterTheOne** [comment]
@bjornlevi you also seem to do work on wasa2il: is wasa2il still actively developed?

**Comment by bjornlevi** [comment]
Yes @PeterTheOne, well. Still being actively maintained and updated.
